### ISSUE TITLE
Allow configuring rate limits

### ISSUE THREAD

**Issue by boosh** [aws/chalice#482]
Any plans to support configuring rate limits for API stages?

**Comment by jamesls** [comment]
Yes, I would love to have this feature in chalice.  We just need to work out an appropriate API for configuring this.
